app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def coopy_file(command: str) -> None:
+    elements = command.split(" ")
```

Comment:
`" "` is default value
```suggestion
    elements = command.split()
```

Reply:
Done